Vim performance state time interval

kbrock opened the pull request:
Moved Helper#get_time_interval to the concept that it represented.

second commit: it also was retrieving a hardcoded value, so avoided the lookup

Fryguy commented on app/models/vim_performance_state.rb:
```diff
@@ -61,7 +61,7 @@ def self.capture(objs)
   def capture
     self.state_data ||= {}
-    self.capture_interval = 3600
+    self.capture_interval = 1.hour.to_i
```
If it's hardcoded, why even store it?

Fryguy replied:
Oh it's a column? - Wild we have this at all.

Fryguy replied:
I'm fine with this for now - future feature just remove the column.

Fryguy commented on app/models/vim_performance_state.rb:
```diff
@@ -162,6 +162,14 @@ def get_assoc(relat, mode = nil)
+  # @param obj object holding vps records
+  # @param timestamp [Time|String] hourly timestamp, prefer Time
+  # @returns [Range] time range
+  def self.get_time_interval(obj, timestamp)
```
obj isn't even used now - can we remove it?

kbrock replied:
This method is only called for hourly.
I think it was written to support all `VimPerformanceState`.
But we only store vim performance state values for hours so it is probably moot.

changing

kbrock commented:
udpate:
- rebased
- dropped obj passed into vps_capture_interval (it is always going to be the same value)
- changed name of variable from capture_interval to date_range.
